qemu/tests/cfg/nested_hyperv_on_kvm.cfg

```diff
@@ -0,0 +1,15 @@
+- nested_hyperv_on_kvm:
```

Comment:
Consider `only x86_64` and `only Windows`

Reply:
Updated, thank you so much for your comment @PaulYuuu